ksml-runner/src/main/java/io/axual/ksml/runner/config/KSMLConfig.java

```diff
@@ -58,9 +60,14 @@ public class KSMLConfig {
     @Builder.Default
     @Getter
     private PrometheusConfig prometheusConfig = DEFAULT_PROMETHEUS_CONFIG;
+
     private String configDirectory;
     private String schemaDirectory;
     private String storageDirectory;
```

Comment:
The `storageDirectory` config is [hard-coded in the helm charts](https://github.com/Axual/ksml/blob/main/packaging/helm-charts/ksml/templates/configmap-runner.yaml#L12). This needs to be made configurable via values.yaml.

Reply:
It's configurable now, and the charts now allow additional claim templates that your can use